src/utils/nonNull.ts

```diff
@@ -36,8 +37,9 @@ export function nonNullOrEmptyValue(
 ): string {
   if (!value) {
     throw new Error(
-      l10n.t('Internal error: Expected value to be neither null, undefined, nor empty') +
-        (propertyNameOrMessage ? `: ${propertyNameOrMessage}` : ''),
+      (errorMessage.internalErrorNullOrUndefinedOrEmpty + propertyNameOrMessage ||
+        `:${propertyNameOrMessage}`) ??
+        '',
```

Comment:
Please revert these ones without `??`. It's a mess to understand now

Reply:
Done

Reply:
@lloydAffinidi did u revert only this function or all?

Reply:
@OleksiiO-affinidi  both `nonNullValue` and `nonNullValueEmptyValue`